docs/adrs/002-Retry_endpoint.md

```diff
@@ -0,0 +1,81 @@
+# Retry Endpoint
+
+Date: 2021-04-27
+
+The current requirement for the Batch/Bulk UI is to be able to retry jobs. Using a job id, it should be possible to get information from the database.
+
+The current implementation of retry is to run jobs using the `run_job` or `run_job_batch` endpoint. This is not adequate due to the following deficiencies:
+
+
+* Lack of book-keeping for the parent job and the child job relationship: Launching a new job will not update the relationship between those jobs
```

Comment:
Make this more specific -- launching a job that should be a child of a given parent job does not update the parent job to add the new child ID (I assume that's the case, anyway)

Reply:
Updated it, thanks